arch/varnish.sysusers

```diff
@@ -0,0 +1,2 @@
+u varnishlog -:varnish "Varnish Cache Proxy"
+g varnish -
```

Comment:
Create this file in the systemd directory and create a symlink, even if arch is the only one to use it so far. The sysusers facility has been on my radar for a while already.

Reply:
aye aye captain